Flea/Shared/ReservationOverview.razor

```diff
@@ -0,0 +1,34 @@
+<div class="grid-container-res">
+    <div class="res-txt">@Name</div>
+    <div class="res-txt">@Stande</div>
+    <div class="res-txt">@(PaymentStatus ? "Betalt" : "Ej betalt")</div>
+    <button class="button res-del-button" @onclick="Delete">Slet</button>
+    <button class="button res-edit-button" @onclick="Edit">Rediger</button>
+</div>
+
+@code {
+    [Parameter]
+    public string Name { get; set; }
+
+    [Parameter]
+    public int Stande { get; set; }

```

Comment:
Again avoid danish words in source code identifiers.

Reply:
fixed